INN/umbrella-citylimits
# Only show special projects in mobile nav if on homepage

## Changes

This pull request makes the following changes:

- Adds a conditional to the mobile nav to only display the special projects menu if on the homepage

Homepage:
![Screen Shot 2019-10-21 at 9 54 57 AM](https://user-images.githubusercontent.com/18353636/67211588-0054d580-f3e9-11e9-98a6-915958e833a6.png)

Any other page:
![Screen Shot 2019-10-21 at 9 55 11 AM](https://user-images.githubusercontent.com/18353636/67211586-0054d580-f3e9-11e9-86af-bf1fa338a8d7.png)

## Why



For #40 / #74 

## Testing/Questions

Features that this PR affects:

- Mobile nav

Steps to test this PR:

1. View the mobile nav on the homepage and verify the special projects menu is there
2. View the mobile nav on **not** the homepage and verify the special projects menu is **not** there
